sockeye/checkpoint_decoder.py

```diff
@@ -129,6 +129,7 @@ def decode_and_evaluate(self,
                                           self.bucket_width_source,
                                           self.bucket_width_target,
                                           inference.LengthPenalty(self.length_penalty_alpha, self.length_penalty_beta),
+                                          0.0,
```

Comment:
maybe we should use keywords in this constructor call to clarify what all of these mean.